stacks/vitess/README.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started with the Vitess Operator
+
+After you have deployed this stack you will have the Vitess operator and an
```

Comment:
It looks like we want to have a section in this README explaining what the Vitess Operator does and its value to the user. Also, the template includes a boilerplate section on getting K8s running on DO. See https://github.com/digitalocean/marketplace-kubernetes/blob/master/docs/LISTING-README-TEMPLATE.md